wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -438,18 +438,26 @@ private[wallet] trait TransactionProcessing extends WalletLogger {
 
         for {
           _ <- insertIncomingTransaction(transaction, totalIncoming)
+
+          addrs <- addressDAO.findAllAddresses()
```

Comment:
Hmm, why is this needed now?

Reply:
To differentiate the spks that belong to the wallet and the ones that don't. For example, we don't want to add the DLC funding output to our spending info db as it will effect our balance, and we could not spend it.